webapp/tbapi/migrations/versions/27935569a4ca_create_notes_table.py

```diff
@@ -0,0 +1,54 @@
+"""create_notes_table
+
+Revision ID: 27935569a4ca
+Revises: a0a902e9fba9
+Create Date: 2018-09-21 20:15:08.405530
+
+"""
+from alembic import op
+import sqlalchemy as sa
+
+
+# revision identifiers, used by Alembic.
+revision = '27935569a4ca'
+down_revision = 'a0a902e9fba9'
+branch_labels = None
+depends_on = None
+
+
+def upgrade(engine_name):
+    globals()["upgrade_%s" % engine_name]()
+
+
+def downgrade(engine_name):
+    globals()["downgrade_%s" % engine_name]()
+
+
+def upgrade_():
+    op.create_table('notes',
+    sa.Column('id', sa.Integer(), nullable=False),
+    sa.Column('patient_id', sa.Integer(), nullable=False),
+    sa.Column('text', sa.String(length=10000), nullable=True),
+    sa.Column('author_id', sa.Integer(), nullable=False),
```

Comment:
Will the `author_id` and `patient_id` ever differ?

If not, can we remove one of them to avoid confusion?

Reply:
will likely become title